packages/components/src/dropdown/stories/index.story.tsx

```diff
@@ -83,3 +85,25 @@ WithNoPadding.args = {
 		</DropdownContentWrapper>
 	),
 };
+
+/**
+ * The `<DropdownContentWrapper>` convenience wrapper can also be used to remove padding entirely,
+ * with a `paddingSize` of `"none"`. This can also serve as a clean foundation to add arbitrary
+ * paddings, for example when child components already have padding on their own.
+ */
+export const WithMenuItems = Template.bind( {} );
+WithMenuItems.args = {
+	...Default.args,
+	renderContent: () => (
+		<DropdownContentWrapper>
```

Comment:
No need for the `DropdownContentWrapper` wrapper since we don't play with the padding here, right?

```suggestion
		<>
```